core/domain/suggestion_services.py

```diff
@@ -357,12 +357,21 @@ def get_reviewable_suggestions(user_id, suggestion_type):
         list(Suggestion). A list of suggestions which the given user is allowed
             to review.
     """
-    return ([
+    all_suggestions = ([
         get_suggestion_from_model(s) for s in (
             suggestion_models.GeneralSuggestionModel
             .get_in_review_suggestions_of_suggestion_type(
                 suggestion_type, user_id))
     ])
+    user_review_rights = user_services.get_user_community_rights(user_id)
+    if suggestion_type == suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT:
+        language_codes = (
+            user_review_rights.can_review_translation_for_language_codes)
+        return [
+            suggestion for suggestion in all_suggestions if (
+                suggestion.change.language_code in language_codes)
```

Comment:
Are the parentheses necessary if the statement is already inside brackets?

Reply:
Parens aren't necessary here -- @DubeySandeep please also deindent this line and the line below by 4 to match the opening bracket indentation. (Thanks @sagangwee for the catch!)

Reply:
Fixed! (I was considering if as a seprate block and use parenthesis to break lines.)